src/bundle/Resources/config/services/factory.yaml

```diff
@@ -4,12 +4,22 @@ services:
         autoconfigure: true
         public: false
 
-    EzSystems\EzRecommendationClient\Factory\:
-        resource: '../../../../src/lib/Factory/*'
+    EzSystems\EzRecommendationClient\Factory\EzRecommendationClientAPIFactory: ~
 
-    EzSystems\EzRecommendationClient\Factory\ExportParametersFactory: ~
+    EzSystems\EzRecommendationClient\Factory\FakeRequestFactory: ~
```

Comment:
Why is this service present in the final bundle configuration? Judging by it's name, it's something used purely for testing?

Reply:
This Factory prepare environment and http request which is necessary to run export in CLI.